### ISSUE TITLE
A11y_VscodeWebviewUiToolkit_AllComponents_ProgressBar_ScreenReader: NVDA does not announces any information about "default progress bar" when navigating using down arrow key in browse mode.

### ISSUE THREAD

**Issue by Shivamd1** [microsoft/vscode-webview-ui-toolkit#187]
### Actual:
NVDA does not announces any information about "default progress bar" when navigating using down arrow key in browse mode.

Similar issue is observed with Jaws

Similar issue is with "Default Progress ring" 

### Expected:
NVDA should be announcing information about "default progress bar" when navigating using down arrow key in browse mode. It should be announcing like "loading"

**GitHubTags:** #A11yMAS;#A11yTCS;#A11ySev2;#SH_VscodeWebviewUiToolkit_Aug2021;#Webview UI Toolkit for Visual Studio Code;#Win32;#FTP;#DesktopApp;#WCAG1.3.1;#NVDA;#Jaws;#STP;#AINotInScope;

### Environment Details:
Application Name: Vscode Webview Ui Toolkit
Windows Version: Win10
Screen Reader: NVDA(2021.1), Jaws(2021.2102.34)

### Repro Steps:

1. Launch VS Code
2. Open WebView UI Toolkit: All Component extension
3. Tab to "Progress bar"
4. Start NVDA in browse mode, use down arrow to navigate to "default progress bar" and check the announcement

### User Impact:
Screen Reader users will not get the proper information about the "default progress bar"

[“Check out Accessibility Insights!](https://accessibilityinsights.io/) - Identify accessibility bugs before check-in and make bug fixing faster and easier.”

**Comment by Shivamd1** [comment]
@hawkticehurst  Could you please provide the latest vsix file so that we can verify it, also could you please reopen the bug, it should only be closed by trusted tester

**Comment by Roopesh-T** [comment]
@daviddossett @hawkticehurst Could you please provide the latest vsix file so that we can verify it, also could you please reopen the bug, it should only be closed by trusted tester

**Comment by daviddossett** [comment]
Same situation here. This component was removed and thus not applicable for testing.

**Comment by Roopesh-T** [comment]
As per the above comment, the code is removed from the library, so closing the bug.
GitHub:#Closed;
![image](https://user-images.githubusercontent.com/86601127/218698283-104bc76d-b5d5-402e-b102-b581a16fe065.png)
![image](https://user-images.githubusercontent.com/86601127/218698287-756d512d-938a-4000-ad3e-a109537a6b08.png)
